deploy/nri-prometheus.tmpl.yaml

```diff
@@ -114,11 +115,19 @@ data:
     # The label used to identify scrapable targets. Defaults to "prometheus.io/scrape".
     scrape_enabled_label: "prometheus.io/scrape"
 
+    # enable_scrape_services Allows to enable scraping the service and not the endpoints behind.
+    # When endpoints are scraped this is no longer needed
```

Comment:
Could we check this in the code? Since it is true by default, it is possible that some users set `enable_scrape_endpoints: true` but forget to disable the old one.

In fact, if the options are mutually exclusive, I'd strip `enable_scrape_services` entirely.

Reply:
I am not totally sure, but I can imagine a possible use-case for which they actually want both.
Otherwise it would be quite obscure that enabling endpoints you lose services